common.sh.in

```diff
@@ -127,6 +127,18 @@ map_disk0() {
 
 unmap_disk0() {
   kpartx -d -p- $1
+
```

Comment:
Uh, why do you reformat the disk in the umap function? This should just unmap any partitions, but not touch the disk.

Reply:
It seems to me that it's right place. I don't want to touch disk until kernel has view of partitions via kpartx, so I opted to put it in unmap. I considered pushing it into CLEANUP, but losetup puts losetup -d in CLEANUP in create and using CLEANUP would involve modifying create also, so it seemed to me like cleaner and smaller patch.